microservices/front-end/src/components/QuestionBank.tsx

```diff
@@ -0,0 +1,211 @@
+import React, { useEffect, useState } from 'react';
+import "./QuestionBank.css";
+
+// Need to define a Question type. ? = optional fields
+interface Question {
+    id: number;
+    title: string;
+    description: string;
+    category: string;
```

Comment:
Perhaps category should be an array of strings? Here is the schema i used for backend:
```
type Question = {
  title: string
  categories: string[]
  complexity: 'Easy' | 'Medium' | 'Hard'
  description: string
};
```

I will change my code to use 'complexity' instead of 'difficulty to be more in line with the question give

Reply:
Ok this one I changed in my new version of code! 😄